### DISCUSSION TITLE
Interaction from builders with snow

### DISCUSSION THREAD

**Discussion by Coocys** [ldtteam/minecolonies#8168] [Q&A]
Hey,
we are currently building a minecolony in a snow biom and we are pretty annoyed that builders always try to remove the snow step by step and not just ignore it. It takes so much longer and issnt really nessesary because you can place a block on/in snow and still works the same. especially if there is a snow storm the snow regenerates and the builder is in a loop until the weather is normal again.

Is this intended or is there a possibility to make this any smoother. Would be nice :D

btw great mod and thank you for you quick responses

**Comment by Raycoms** [comment]
The problem is always that, if we don't remove it, there is a chance that there is snow left inside of buildings.

**Reply by Coocys** [reply]
Is there any way to speed up the snow removal maby? Except better equipment.

**Reply by wakkytabbaky** [reply]
researching the block destruction tech + high atheletics if that doesnt tickle your fancy enough, can speed up the block breaking via configs also
